models.py

```diff
@@ -1,32 +1,100 @@
+from sqlmodel import Field, SQLModel, Relationship
+from typing import List, Optional
 from datetime import datetime
-from sqlmodel import Field, SQLModel
 
-class PostBase(SQLModel):
+class ProfileBase(SQLModel):
+    username: str = Field()
+    email: str = Field()
+    name: str = Field()
     description: str = Field()
-    transaction: float = Field()
+    is_formalized: bool = Field(default=False)
+    start_year: int = Field(default=0)
+    contact_phone: str = Field()
+    instagram_link: Optional[str] = Field(default=None)
+    x_link: Optional[str] = Field(default=None)
+    facebook_link: Optional[str] = Field(default=None)
+    pix_qr_code_link: Optional[str] = Field(default=None)
+    site: Optional[str] = Field(default=None)
 
-class Post(PostBase, table=True):
+# Profile model with relationships
+class Profile(ProfileBase, table=True):
     id: int | None = Field(default=None, primary_key=True)
     created_at: datetime = Field(default_factory=datetime.now)
     ong: int = Field(nullable=False)
+    
+    # Relationships
+    gallery_images: List["GalleryImage"] = Relationship(back_populates="profile")
+    skills: List["Skill"] = Relationship(back_populates="profile")
+    causes: List["Cause"] = Relationship(back_populates="profile")
+    sustainable_development_goals: List["SDG"] = Relationship(back_populates="profile")
```

Comment:
Já que as 4 tabelas que citei no outro comentário não vão ser tabelas da banco dados, sugiro fazer as seguintes alterações:
```suggestion
    gallery_images: Optional[List["GalleryImage"]] = Field(default=None)
    skills: Optional[List["Skill"]] = Field(default=None)
    causes: Optional[List["Cause"]] = Field(default=None)
    sustainable_development_goals: Optional[List["SDG"]] = Field(default=None)
```
Atenção: não sei se (por exemplo) `List["Skill"]` é válido se `Skill` não é uma tabela no banco de dados. Talvez sim. Talvez não. Também, tem que importar o `Optional` de `typing`. O uso de `Optional` aqui é porque eu suspeito que algumas ONGs podem não ter esses campos.